lib/fetch/headers.js

```diff
@@ -301,14 +301,10 @@ class Headers {
 
     // 6. If this’s header list does not contain name, then
     //    return.
-    if (!this[kHeadersList].contains(name)) {
-      return
-    }
-
```

Comment:
Please keep this?

Reply:
Since there seemed to be no side effects, I did not consider it necessary.

Reply:
I think this change is not the purpose of this PR, so I will make the change in another PR.